cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/KafkaCruiseControl.java

```diff
@@ -251,7 +252,8 @@ public static void sanityCheckCapacityEstimation(boolean allowCapacityEstimation
    * @param brokerIds The broker ids.
    * @param dryRun Whether it is a dry run or not.
    * @param throttleAddedBrokers Whether throttle the brokers that are being added.
-   * @param goals The goals to be met when adding the brokers. When empty all goals will be used.
+   * @param goals The goal names (i.e. each matching {@link Goal#name()}) to be met when adding the brokers.
+   *              When empty all goals will be used.
```

Comment:
Same Javadoc update needs to be applied to `rebalance()` and `ignoreProposalCache()` method of this Class.